src/Aspire.Hosting.Kafka/KafkaBuilderExtensions.cs

```diff
@@ -25,11 +27,34 @@ public static IResourceBuilder<KafkaServerResource> AddKafka(this IDistributedAp
         var kafka = new KafkaServerResource(name);
         return builder.AddResource(kafka)
             .WithEndpoint(targetPort: KafkaBrokerPort, port: port, name: KafkaServerResource.PrimaryEndpointName)
+            .WithEndpoint(targetPort: KafkaInternalBrokerPort, name: KafkaServerResource.InternalEndpointName)
             .WithImage(KafkaContainerImageTags.Image, KafkaContainerImageTags.Tag)
             .WithImageRegistry(KafkaContainerImageTags.Registry)
             .WithEnvironment(context => ConfigureKafkaContainer(context, kafka));
     }
 
+    /// <summary>
+    /// Adds a Kafka UI container to the application. This version of the package defaults to the 0.7.2 tag of the provectuslabs/kafka-ui container image.
+    /// </summary>
+    /// <param name="builder"></param>
+    /// <param name="port"></param>
+    /// <param name="containerName"></param>
+    /// <returns></returns>
+    public static IResourceBuilder<KafkaServerResource> WithKafkaUI(this IResourceBuilder<KafkaServerResource> builder, int? port = null, string? containerName = null)
```

Comment:
Ah I just realized this is missing the overload that exposes the `KafkaUIContainerResource` that allows others to change container properties (tag etc). https://github.com/dotnet/aspire/blob/f552a0d94409f0e605a253a38f93da3a5be6b215/src/Aspire.Hosting.PostgreSQL/PostgresBuilderExtensions.cs#L97

Reply:
Updated.

@davidfowl `WithPgAdmin` have a nice capability which allows to use a single instance to manage multiple resources. Should we implement something similar to manage several Kafka resources in a unique KafkaUI container ?
`KafkaBuilderExtensions` and related resources live in `Aspire.Hosting` namespace. I noticed this is not consistent accross components (see MongoDB or Postgres resources for example). What is the correct namespace rule ?

Reply:
Yes we should. Extensions should all live in the Aspire.Hosting namespace.

cc @mitchdenny

Reply:
done